app/src/main/java/com/randomappsinc/studentpicker/choosing/NameChoosingHistoryManager.java

```diff
@@ -0,0 +1,55 @@
+package com.randomappsinc.studentpicker.choosing;
+
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+
+import com.afollestad.materialdialogs.MaterialDialog;
+import com.randomappsinc.studentpicker.R;
+import com.randomappsinc.studentpicker.models.ListInfo;
+import com.randomappsinc.studentpicker.utils.NameUtils;
+import com.randomappsinc.studentpicker.utils.UIUtils;
+
+public class NameChoosingHistoryManager {
+
+    public interface Listener {
```

Comment:
Setting up a listener just to show a Toast is sort of underwhelming, especially when this class already shows a Toast in another use case. To minimize logic in the fragment, let's just show the Toast in this class. The dialog always has access to a context since it's a fragment with views.